setup.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import absolute_import, division, print_function
 
+import setuptools
```

Comment:
Note that you have to monkey patch w/ setuptools if you use `install_requires` or `entry_points`.

Without it, those options are simply ignored.